random_lookahead.smt2

```diff
@@ -0,0 +1,4 @@
+(set-option :pure-lookahead true)
```

Comment:
These `smt2` files should probably not be part of the PR, no?

Reply:
Sure